lib/matplotlib/colors.py

```diff
@@ -1739,7 +1739,7 @@ def __call__(self, value, clip=None):
         else:
             max_col = self.Ncmap
         # this gives us the bins in the lookup table in the range
-        # [0, _n_regions - 1]  (the offset is baked in in the init)
+        # [0, _n_regions - 1]  (the offset is baked in the init)
```

Comment:
This is not a typo.

Reply:
Prob should be `baked-in in the init`

Reply:
fixing this in a follow-up PR since this one was merged before I could address it.